packages/shared/src/browser.ts

```diff
@@ -1,3 +1,7 @@
+/**
```

Comment:
I added JSDoc comments to this file because at the beginning I thought: Yeah, if I move these files I can "quickly" add those, too. But it got out of hand and would have polluted the changed files, so I stopped after this file